clients/restore/restore.go

```diff
@@ -370,7 +370,7 @@ func (c Client) restoreOrg(ctx context.Context, name string) (string, error) {
 	return *orgs.GetOrgs()[0].Id, nil
 }
 
-func bucketToDBI(b br.ManifestBucketEntry) br.DatabaseInfo {
+func bucketToDBI(b br.ManifestBucketEntry) *br.DatabaseInfo {
```

Comment:
Why do these now need to return pointers?

Reply:
To both: with the new protobuf library, leaving those as before would cause issues with `go vet`, stating `return copies lock value` as `proto.MessageState` contains a mutex now